src/components/Person/Person.jsx

```diff
@@ -1 +1,24 @@
-// export const Person = ({ person }) => ();
+import React from 'react';
+
+export const Person = ({ person: {
+  name, age, sex, isMarried, partnerName,
+} }) => {
+  const partner = (sex === 'm') ? 'wife' : 'husband';
+
+  return (
+    <section className="Person">
+      <h2 className="Person__name">
+        My name is
+        {` ${name}` }
+      </h2>
+      {age && (
+        <p className="Person__age">
+          I am
+          {' '}
+          {age}
```

Comment:
Make a template string from this.

[More info](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals)